types/node/inspector.d.ts

```diff
@@ -0,0 +1,2484 @@
+// Type definitions for inspector (built-in module in Node 8+)
```

Comment:
This shouldn't have its own header. Just one for the whole package in `index.d.ts`.